### PR TITLE
ReactJS setup with RubyOnRails App

### PR DESCRIPTION
**Pull request by nexondevs**
# Setting up ReactJs, with RubyOnRails

## Working on `ROR`:
- Added `react-rails` gem.

## Working on `ReactJS`:
- Added `react` to `package.json`
- Added `react-dom` to `package.json`
- Added `react_ujs` to `package.json`

### CODE REVIEW

**Review by in-techouse — CHANGES_REQUESTED** [review]
Delete the file `app/javascript/components/.keep`

And add all `.keep` files to gitignore

**Comment by in-techouse** [comment]
Delete the file `app/javascript/packs/hello_react.jsx`

File: config/webpacker.yml [hunk 1]
```diff
@@ -33,6 +33,7 @@ default: &default
     - .woff2
 
   extensions:
+    - .jsx
```
Comment (in-techouse) [inline config/webpacker.yml]: We will use the `.js` extension.
Remove the `.jsx`.

File: Gemfile [hunk 2]
```diff
@@ -28,6 +30,7 @@ gem 'jbuilder', '~> 2.7'
 # Reduces boot times through caching; required in config/boot.rb
 gem 'bootsnap', '>= 1.4.4', require: false
 
+gem 'react-rails'
```
Comment (in-techouse) [inline Gemfile]: Add version with `=`

**Review by in-techouse — CHANGES_REQUESTED** [review]

File: config/webpacker.yml (hunk 1 above)
Comment (in-techouse) [reply config/webpacker.yml]: @Sajidalirordevloper

File: Gemfile (hunk 2 above)
Comment (in-techouse) [reply Gemfile]: @Sajidalirordevloper

File: config/webpacker.yml (hunk 1 above)
Comment (nexondevs) [reply config/webpacker.yml]: @in-techouse  jsx extension change to js

**Pull request merged**